js/functions.js

```diff
@@ -2211,3 +2213,109 @@ function getEventDescription(isAerobatics, locationName, minutes) {
     var desc = 'יחל ב';
     return `${desc}${locationName} בעוד ${minutes} דקות`;
 }
+
+
+
+(function () {
+    var userLoc = null;
+    navigator.geolocation.watchPosition(function(newLoc){
+        userLoc = newLoc;
+        userLoc = {lon: userLoc.coords.longitude, lat: userLoc.coords.latitude};
+    });
+
+    var aircraftsName = [       
```

Comment:
This info already loaded from  aircraftinfo.json file, there is no need for that conversation table. Take it from the aircraft object itself.

Reply:
We will check that.
@mayPeleg Call me, lets figure it out.